packages/@aws-cdk/assertions/lib/template.ts

```diff
@@ -152,16 +152,23 @@ export class Template {
    * Use the `Match` APIs to configure a different behaviour.
    */
   public findMappings(logicalId: string, props: any = {}): { [key: string]: { [key: string]: any } } {
-    return findMappings(this.inspector, logicalId, props);
+    return findMappings(this.template, logicalId, props);
   }
 
   /**
    * Assert that the CloudFormation template matches the given value
    * @param expected the expected CloudFormation template as key-value pairs.
    */
-  public templateMatches(expected: {[key: string]: any}): void {
-    const assertion = assert.matchTemplate(expected);
-    assertion.assertOrThrow(this.inspector);
+  public templateMatches(expected: any): void {
+    const matcher = Matcher.isMatcher(expected) ? expected : Match.objectLike(expected);
```

Comment:
Since `templateMatches()` now defaults to the `objectLike()` matcher, does that mean that there is a behavioral breaking change in addition to the method signature? I think we should document that in the PR description also.

Reply:
Good call. Updated.